src/fiatconnect/fiatAccountSchemas/mobileMoney.ts

```diff
@@ -0,0 +1,60 @@
+import { FiatAccountSchema, FiatAccountType } from '@fiatconnect/fiatconnect-types'
+import { getObfuscatedAccountNumber } from 'src/fiatconnect'
+import { FiatAccountFormSchema } from 'src/fiatconnect/fiatAccountSchemas/types'
+import i18n from 'src/i18n'
+
+export const getMobileMoneySchema = (implicitParams: {
+  country: string
+  fiatAccountType: FiatAccountType
+}): FiatAccountFormSchema<FiatAccountSchema.MobileMoney> => {
+  const mobileValidator = (input: string) => {
+    const regex = /^\+\d{3,15}$/
```

Comment:
Can you put a code comment next to this explaining what it means? It looks like any string starting with + followed by 3 to 15 digits, is that right? If so, seems like 3 digits is too short

Reply:
also we should use [0-9] and not \d which includes Eastern Arabic numerals

Reply:
Yes thats right, but there doesn't seem to be an explicit minimum digit length from the official international numbering [doc](https://www.itu.int/rec/T-REC-E.164-201011-I/en). I'll increase it a bit though.